filesharer.py

```diff
@@ -0,0 +1,12 @@
+from filestack import Client
+
+
+class FileSharer:
+    def __init__(self, filepath, api_key="AW6dnnV2cS2Gl0vhvqZKWz"):
```

Comment:
There is a pattern which name is `config`. I offer to create a file with name `config.py` and put api_key in this file.